app/config/FrontendAppConfig.scala

```diff
@@ -26,30 +26,30 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 import java.time.LocalDate
 
 @Singleton
-class FrontendAppConfig @Inject() (
-  configuration: Configuration,
-  val servicesConfig: ServicesConfig
-) {
-
-  val host: String           = configuration.get[String]("host")
-  val appName: String        = configuration.get[String]("appName")
-  lazy val mfaUpliftUrl      = configuration.get[String]("urls.mfaUplift")
+class FrontendAppConfig @Inject()(
+                                   configuration: Configuration,
```

Comment:
try to avoid lots of white space changes, it dilutes the actual changes of the pr :)

Reply:
autoformat strikes again 😅